Gemfile

```diff
@@ -41,16 +41,13 @@ gem "pg_search"
 gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 
 gem "cloudinary"
```

Comment:
Why did you delete those gems?

Reply:
When I was running bundle install, there was a warning that some gems were listed more than once which may create problems in the future, so I just deleted the duplicates.